src/main/g8/src/main/resources/META-INF/squbs-meta.conf

```diff
@@ -0,0 +1,23 @@
+cube-name = $organization$.$project$.$appname$
+
+cube-version = "0.1.0-SNAPSHOT"
```

Comment:
I would use the supplied version value here instead of hard coding it here as well.